docs/src/api/class-electronapplication.md

```diff
@@ -135,6 +135,13 @@ Convenience method that waits for the first application window to be opened.
   // ...
 ```
 
+### param: ElectronApplication.firstWindow.options
+* since: v1.33
+* langs: js
+- `options` ?<[Object]>
+  - `timeout` ?<[float]> maximum time to wait for in milliseconds. Defaults to `30000` (30 seconds). Pass `0` to
+    disable timeout. The default value can be changed by using the [`method: BrowserContext.setDefaultTimeout`].
```

Comment:
BrowserContext.setDefaultTimeout?

Reply:
what about it?

Reply:
There is no browsercontext in electron?

Reply:
What about https://playwright.dev/docs/next/api/class-electronapplication#electron-application-context ?

Reply:
You won